samples/ResourceOwnerPasswordFlow/Certificate.keys

```diff
@@ -0,0 +1,8 @@
+﻿p4qYnZxKxntusUtZWayPMGIy1YxxIji6jl0/Nc6WEu/iOhXlCVBId2QPzVQLlFKbJFKE6yb0gQnL2CUY1gZLXrVvAtdn7mla2VJtjDqUHi83D1OnEBvpcdzdNz6DPYGOwa64WBDP0amUAees6uZKdXp6RCb+vN109CE/gW9tbsXxbkjhxMsr9LXCLqpYSwkYZyxXBH/qaV7sCgv3qkNWmsnuAMXE+ztDdVY6zLtXGgkkmrWNk09siFIERgi6QJwOpf93RRc6LQG/HNCwz30sUroVMqACZPJQyQgT2I5HUh2AiTDDKg7kE1maR78+FhvLv/XkVXrwpPVe1f8tJX63wQ==
```

Comment:
Will do :) I may have some time for this in the near future, because we've just delivered the two proposals that were taking up all my time.